pkg/dashboard/ttlcontroller/ttlcontroller.go

```diff
@@ -53,7 +51,9 @@ func NewController(
 	event core.EventStore,
 	ttlc *TTLconfig,
 ) *Controller {
+	logger, _ := z.NewDefaultZapLogger()
```

Comment:
please do not create another logger, acquire the logger as a parameter in the constructor.